docs/Configuration.md

```diff
@@ -378,9 +378,9 @@ array of absolute paths to additional locations to search when resolving
 modules. Use the `<rootDir>` string token to include the path to your project's
 root directory. Example: `["<rootDir>/app/"]`.
 
-### `notify` [boolean]
+### `notify` [string]
```

Comment:
I'm not sure we want this to overload. Maybe a separate `notify-mode` or something? Changing its type is a breaking change, and would mean we couldn't merge this PR for months

Something like what travis gives you, where you can get a notification on failure, and when that turns green, but not green after one another. (https://docs.travis-ci.com/user/notifications/#Changing-notification-frequency)

The you could have `onFailure`, `onSuccess`, `onChange`. for instance.

Reply:
That sounds like a better design. I like the additional events travis has as well.